galaxy_ng/app/access_control/statements/insights.py

```diff
@@ -385,70 +70,53 @@
             "action": ["list"],
             "principal": "authenticated",
             "effect": "allow",
-            "condition": ["has_model_perms:galaxy.view_synclist", "has_rh_entitlements"],
+            "condition": ["has_model_perms:galaxy.view_synclist"],
         },
         {
             "action": ["retrieve"],
             "principal": "authenticated",
             "effect": "allow",
-            "condition": ["has_model_or_obj_perms:galaxy.view_synclist", "has_rh_entitlements"],
+            "condition": ["has_model_or_obj_perms:galaxy.view_synclist"],
         },
         {
             "action": ["destroy"],
             "principal": "authenticated",
             "effect": "allow",
-            "condition": ["has_model_perms:galaxy.delete_synclist", "has_rh_entitlements"],
+            "condition": ["has_model_perms:galaxy.delete_synclist"],
         },
         {
             "action": ["create"],
             "principal": "authenticated",
             "effect": "allow",
-            "condition": ["has_model_perms:galaxy.add_synclist", "has_rh_entitlements"],
+            "condition": ["has_model_perms:galaxy.add_synclist"],
         },
         {
             "action": ["update", "partial_update"],
             "principal": "authenticated",
             "effect": "allow",
-            "condition": ["has_model_perms:galaxy.change_synclist", "has_rh_entitlements"],
+            "condition": ["has_model_perms:galaxy.change_synclist"],
         },
     ],
     "MySyncListViewSet": [
         {
             "action": ["retrieve", "list", "update", "partial_update", "curate"],
             "principal": "authenticated",
             "effect": "allow",
-            "condition": ["has_rh_entitlements", "is_org_admin"],
-        },
-    ],
-    "TaskViewSet": [
-        {
-            "action": ["list", "retrieve"],
-            "principal": "authenticated",
-            "effect": "allow",
-        },
-    ],
-    "TagViewSet": [
-        {
-            "action": ["list", "retrieve"],
-            "principal": "authenticated",
-            "effect": "allow",
+            "condition": ["is_org_admin"],
         },
     ],
     "GroupViewSet": [
         {
             "action": ["list", "retrieve"],
             "principal": "authenticated",
             "effect": "allow",
-            "condition": "has_rh_entitlements",
         },
     ],
-    "DistributionViewSet": _deny_all,
     "MyDistributionViewSet": [
         {
             "action": ["list", "retrieve"],
             "principal": "authenticated",
             "effect": "allow",
-            "condition": "has_rh_entitlements",
         },
     ],
     "ContainerRepositoryViewSet": _deny_all,
```

Comment:
We have a deny on ContainerRepositoryViewSet... the current insights.py does not have deny on ContainerRegistryRemoteViewSet, ContainerRemoteViewSet, and ContainerReadmeViewset. But since we are inheriting standalone which has some things allowed for authenticated users, we probably should put deny on them.

Reply:
I think we can also deny `LegacyAccessPolicy` and `pulp_ansible/v3/legacy-redirected-viewset` since they also allow access for all authenticated users. But otherwise the diff looks good.